core/src/epicli/data/common/ansible/playbooks/roles/filebeat/templates/filebeat.yml.j2

```diff
@@ -93,6 +93,11 @@ filebeat.inputs:
   {{ k }}: {{ v }}
   {% endfor %}
   {% endif %}
+- type: log
+  enabled: true
+  paths:
+    - /var/log/pgbouncer/*.log*
+  exclude_files: [".gz$"]
 {% endif %}
```

Comment:
Please add a posibility to use multiline for pgbouncer as well. Dont forget about update [docs](https://github.com/epiphany-platform/epiphany/blob/develop/docs/home/howto/LOGGING.md#how-to-add-multiline-support-for-filebeat-logs). Thanks.

Reply:
Pgbouncer logs do not seem to generate multilines. Do you think it is neccessary anyways then?

Reply:
Hmm I don't know the structure of pgbouncer logs. It would be good to have this as an option (its not enabled by default) as we already provided it for postgresql logs. So maybe let's involve someone in discussion.

Reply:
Hmm I don't know the structure of pgbouncer logs. It would be good to have this as an option (not enabled by default) as we already provided it for postgresql logs. So maybe let's involve someone else in discussion.

Reply:
I've checked on Ubuntu, there is no any multi-line entry in log, even after changing `verbose = 0` to `verbose = 2`.